web/router.ex

```diff
@@ -55,6 +55,8 @@ defmodule CodeCorps.Router do
 
     post "/webhooks/stripe/connect", StripeConnectEventsController, :create
     post "/webhooks/stripe/platform", StripePlatformEventsController, :create
+    post "/webhooks/github_issues", GithubIssueController, :create
+    post "/webhooks/github_issue_comments", GithubIssueCommentController, :create
```

Comment:
How do you feel about calling these actions `:handle`. I feel `:create` is a bit misleading here.